### PR TITLE
Copied and modified the acquisition costs page.

### PR DESCRIPTION
**Pull request by DavidWoodward37**

### CODE REVIEW

**Comment by miranda-hawkes** [comment]
please fix merge conflicts (if there actually are any!)

File: test/routes/shares/RoutesSpec.scala [hunk 1]
```diff
@@ -156,4 +156,18 @@ class RoutesSpec extends UnitSpec with WithFakeApplication with Matchers {
       path shouldEqual "/calculate-your-capital-gains/resident/shares/personal-allowance"
     }
   }
+
+  "The URL for the resident shares acquisitionCosts action" should {
+    "be equal to /calculate-your-capital-gains/resident/shares/acquisition-costs" in {
+      val path = controllers.resident.shares.routes.GainController.acquisitionCosts().toString()
```
Comment (miranda-hawkes) [inline test/routes/shares/RoutesSpec.scala]: You can shorten this to:
_val path = GainController.acquisitionCosts().toString()_

File: test/routes/shares/RoutesSpec.scala [hunk 2]
```diff
@@ -156,4 +156,18 @@ class RoutesSpec extends UnitSpec with WithFakeApplication with Matchers {
...
+      path shouldEqual "/calculate-your-capital-gains/resident/shares/acquisition-costs"
+    }
+  }
+
+  "The URL for the resident shares submitAcquisitionCosts action" should {
+    "be equal to /calculate-your-capital-gains/resident/shares/acquisition-costs" in {
+      val path = controllers.resident.shares.routes.GainController.submitAcquisitionCosts().toString()
```
Comment (miranda-hawkes) [inline test/routes/shares/RoutesSpec.scala]: You can shorten this to:
_val path = GainController.submitAcquisitionCosts().toString()_ 

Could you also drop these route tests below acquisition value and put a _/* Acquisition Costs routes */_ comment please :)

File: test/views/resident/properties/gain/AcquisitionCostsViewSpec.scala [hunk 3]
```diff
@@ -56,6 +56,10 @@ class AcquisitionCostsViewSpec extends UnitSpec with WithFakeApplication with Fa
       }
     }
 
+    "have a home link to 'home-link'" in {
+      doc.getElementById("homeNavHref").attr("href") shouldEqual controllers.resident.properties.routes.GainController.disposalDate().toString()
```
Comment (miranda-hawkes) [inline test/views/resident/properties/gain/AcquisitionCostsViewSpec.scala]: Please change from properties to shares

File: test/views/resident/shares/gain/AcquisitionCostsViewSpec.scala [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+
+      "have the correct back link text" in {
+        backLink.text shouldBe commonMessages.calcBaseBack
+      }
+
+      "have the back-link class" in {
+        backLink.hasClass("back-link") shouldBe true
+      }
+
+      "have a link to Acquisition Value" in {
```
Comment (miranda-hawkes) [inline test/views/resident/shares/gain/AcquisitionCostsViewSpec.scala]: Here, could you either pass in a real route or amend the test name to reflect what it is actually asserting

File: test/views/resident/properties/gain/AcquisitionCostsViewSpec.scala (hunk 3 above)
Comment (DavidWoodward37) [inline test/views/resident/properties/gain/AcquisitionCostsViewSpec.scala]: This is the properties action spec though?

File: conf/resident_shares.routes [hunk 5]
```diff
@@ -21,6 +21,11 @@ POST    /acquisition-value                controllers.resident.shares.GainContro
 
 #Acquisition Costs Routes
 GET     /acquisition-costs                controllers.resident.shares.GainController.acquisitionCosts
+POST    /acquisition-costs                controllers.resident.shares.GainController.submitAcquisitionCosts
+
+#Summary Routes
+GET     /summary                          controllers.resident.shares.SummaryController.summary
```
Comment (miranda-hawkes) [inline conf/resident_shares.routes]: summary action is duplicated, could you remove this please

File: test/views/resident/properties/gain/AcquisitionCostsViewSpec.scala (hunk 3 above)
Comment (miranda-hawkes) [inline test/views/resident/properties/gain/AcquisitionCostsViewSpec.scala]: Ignore me :+1:
